### PR TITLE
Drop yast2-users-devel-doc package

### PR DESCRIPTION
**Pull request by imobach**
Drop yast2-users-devel-doc package ([fate#320356](https://fate.suse.com/320356)). Replaces #101.

### CODE REVIEW

File: package/yast2-users.spec [hunk 1]
```diff
@@ -128,8 +119,4 @@ provided by yast2-users package.
 %doc %{yast_docdir}/TODO.txt
```
Comment (jreidinger) [inline package/yast2-users.spec]: To be honest I am not sure if it is good idea to distribute TODO documentaiton.

**Comment by jreidinger** [comment]
just NP, LGTM

**Comment by imobach** [comment]
@jreidinger I agree. I've removed that documentation.

**Comment by jreidinger** [comment]
LGTM

**Comment by imobach** [comment]
Thanks!

**Pull request merged**